tests/functional/conftest.py

```diff
@@ -1,100 +1,354 @@
 from tests.scripts.pds_request import GenericPdsRequestor, PdsRecord
 import pytest
-
+from pytest_bdd import given, when, then, parsers
+from jsonpath_rw import parse
+from pytest_check import check
+import uuid
+import urllib
 from .utils.apigee_api_apps import ApigeeApiDeveloperApps
 from .utils.apigee_api_products import ApigeeApiProducts
-from .config_files import config
+from .configuration import config
+from .configuration.config import BASE_URL, PDS_BASE_PATH
 import random
+from requests import Response, get, patch
 import json
 from typing import Union
-
+from .data.searches import Search
+from .data.updates import Update
+from .data import searches
+from .data import updates
+from .data import patients
+from .data.patients import Patient
+from .data.responses import RESPONSES
+from copy import copy
 
 from pytest_nhsd_apim.identity_service import (
     ClientCredentialsConfig,
     ClientCredentialsAuthenticator,
 )
 
 from pytest_nhsd_apim.apigee_apis import (
-    ApigeeClient,
     ApiProductsAPI,
-    ApigeeNonProdCredentials,
     DeveloperAppsAPI,
 )
 
 import logging
 
 LOGGER = logging.getLogger(__name__)
-DEVELOPER_EMAIL = "apm-testing-internal-dev@nhs.net"
-
-
-@pytest.fixture()
-def client():
-    config = ApigeeNonProdCredentials()
-    return ApigeeClient(config=config)
 
 
 @pytest.fixture()
-def api_products(client):
-    return ApiProductsAPI(client=client)
```

Comment:
There is already a fixture in pytest-nhsd-apim's library called `products_api`. And although defined at the session scope, this fixture, as it currently stands is not needed at the default function scope.

Reply:
no issue with removing fixture - was the LOGGER not used? Would logging be useful?